Monika After Story/game/script-ch30.rpy

```diff
@@ -1126,6 +1126,9 @@ label ch30_post_mid_loop_eval:
 #                ):
 #                pushEvent("monika_battery")
 
+        # Resets Monika's I love you reaction to its default
+        $ persistent._mas_monika_love_reaction_event = 0
```

Comment:
Okay, some things I'm noticing. Why is this persistent? This shouldn't need to be saved over launches since you're resetting it anyway. That said as well, why use an integer value instead of a boolean?

Also, as a side note, and maybe it's not too much of an issue, but the problem I kinda have with this, looking at it is that it's not expandable to multiple greets without more variables.

Reply:
I did not think about this, it does not need to be persistent.